packages/react/src/components/Cta/README.md

```diff
@@ -0,0 +1,54 @@
+# CTA
+
+> The CTA component will be used to select different cta types pages.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/components/imagecomponent/imagecomponent';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'yourapplication.scss';
+
+function App() {
+  return <CTA style={style} type={type} {...cta} />;
+}
```

Comment:
I don't think an adopter can just copy and paste this into their application and get it working. `cta` isn't a prop.

```javascript
function App() {
  return <CTA
    style='text'
    type='local'
    copy='IBM Homepage'
    href='www.ibm.com'
  />;
}
```